incubator-tools/cer_wer/cer_wer.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>Provide more description of what these are.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>This is taken care, adding the description here also <code>The objective of the tool is to evaluate the character error rate which measures the rate of errors at the character level in the generated output compared to the reference. and word error rate which measures the rate of errors at the word level in the generated output compared to the reference.Both CER and WER can be used to provide a quantitative measure of accuracy, with lower values indicating better performance.</code></p>